scripts/webpack/browser.js

```diff
@@ -76,15 +73,17 @@ module.exports = ({ isLegacyJS }) => ({
         rules: [
             {
                 test: /\.[jt]sx?|mjs$/,
-                exclude: [
-                    {
-                        test: /node_modules/,
-                        exclude: [
-                            /@guardian\/(?!(automat-modules))/,
-                            /dynamic-import-polyfill/,
-                        ],
-                    },
-                ],
+                exclude: {
+                    and: [/node_modules/],
+                    not: [
+                        // Include all @guardian modules, except automat-modules
```

Comment:
Unless I'm confused (v possible) shouldn't this read // Exclude all..., not Include

Reply:
`exclude not` means `include`.  Tbh, this was harder to understand and test than the regex, so if I can do it a simpler way, for sure, I'd like to!